bbot/modules/wpscan.py

```diff
@@ -30,13 +30,20 @@ class wpscan(BaseModule):
         "disable_tls_checks": "Disables the SSL/TLS certificate verification (Default True)",
         "force": "Do not check if the target is running WordPress or returns a 403",
     }
+    deps_apt = ["curl", "build-essential"]
```

Comment:
Since `build-essential` is a debian-specific metapackage, we should probably break this out into `gcc` and `make`.

Reply:
Ah good catch